### ISSUE TITLE
Should skip missing data in parcoords

### ISSUE THREAD

**Issue by archmoj** [plotly/plotly.js#4341]
Reported in https://community.plot.ly/t/plotly-graph-objects-parcoords-nan-replaced-by-0/29725
`If some of the columns contain nan during the display of plotly.graph_objects.Parcoords the data are displayed as 0, it will be nice to don’t to treat the value as a nan and not as 0`.

[demo](https://codepen.io/MojtabaSamimi/pen/oNNyWPm?editors=0010)

**Comment by etpinard** [comment]
I'm not sure about this. I think we should still draw the dimension axis and simply not draw the line.

**Comment by alexcjohnson** [comment]
Agreed - the ideal behavior would be to *just* remove the line segments to and from the `nan` value, but continue to show other segments of the same object.

**Comment by archmoj** [comment]
What would happen with selection then?

**Comment by alexcjohnson** [comment]
I'd say if the dimension with `nan` has any constraint on it at all, this object should be filtered out, but if it's unconstrained then this object should remain selected.
